src/ui/FileViewer/ToggleHeader/ToggleHeader.jsx

```diff
@@ -6,43 +6,29 @@ import Title, { TitleCoverage, TitleFlags, TitleHitCount } from './Title/Title'
 
 function ToggleHeader({
   title,
-  flagNames,
-  onFlagsChange,
-  coverageIsLoading,
   sticky = false,
   showHitCount = false,
+  showFlagsSelect = false,
 }) {
-  /**
-   * Header component that toggles covered, partial and uncovered lines for the File Viewer page.
-   * This component can also filter coverage by flag name
-   * @param {Object} flagData flag names and their respective state and setter
-   * @param {Bool} coverageIsLoading loading boolean to conditioanlly render the flags component
-   */
-
   return (
     <Title title={title} sticky={sticky}>
-      <TitleCoverage coverage={LINE_STATE.UNCOVERED} />
-      <TitleCoverage coverage={LINE_STATE.PARTIAL} />
-      <TitleCoverage coverage={LINE_STATE.COVERED} />
-      {flagNames && flagNames?.length > 1 && (
-        <TitleFlags
-          flags={flagNames}
-          onFlagsChange={onFlagsChange}
-          flagsIsLoading={coverageIsLoading}
-        />
-      )}
-      <TitleHitCount showHitCount={showHitCount} />
+      <div className="flex flex-row gap-2">
```

Comment:
Regarding my col comment above, couldn't this be left without a div, and the Title children using `flex-row`?

Reply:
No this needs to be wrapped into it's own div or else it will break because of the responsive changes made in the prior comment.